pkgs/by-name/co/codecov-cli/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "codecov-cli";
+  version = "9.0.4";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
and set `build-system = [ setuptools ]` to avoid the legacy build.